testpilot/frontend/static-src/app/views/landing-page.js

```diff
@@ -29,6 +36,21 @@ export default PageView.extend({
       }),
         '[data-hook="experiment-list"]');
     }
+
+    app.once('webChannel:addon-self:installed', () => {
+      document.querySelector('.button.default').classList.remove('state-change');
+      const wrapper = this.query('.centered-banner');
+      wrapper.innerHTML = installedMsgTmp;
+      wrapper.querySelector('.button').onclick = () => {
+        app.webChannel.sendMessage('hide-installed-panel', {});
+      };
+    });
+  },
+
+  installClicked() {
+    const btn = this.query('.button.default');
+    btn.classList.add('state-change');
+    btn.querySelector('span').textContent = 'Installing Test Pilot...';
```

Comment:
Nope, this should probably instead be a hidden element that gets revealed, with content marked with `data-l10n-id` and a corresponding string in `app.l20n`. We can't directly set text content like this with L20n